Interoperability/python/src/qsharp/chemistry.py

```diff
@@ -0,0 +1,170 @@
+            return NotImplemented
+        return self.__dict__ == other.__dict__
+
+    def load_fermion_hamiltonian(self, index_convention = 'UpDown') -> FermionHamiltonian:
+        """
+        Loads the fermion hamiltonian associated with this electronic structure problem.
+
+        `index_convention` can be 'UpDown' or 'HalfUp'
+        """
+        logger.info(f"Loading fermion hamiltonian from problem description using index_convention '{index_convention}'.")
+        args = { 'problem_description': self.__dict__, 'index_convention': index_convention }
+        args_json = json.dumps(map_tuples(args))
+        data = qsharp.client._execute(f'%chemistry.fh.load {args_json}', raise_on_stderr=True)
+        return FermionHamiltonian(data)
+
+        
+    def load_input_state(self, wavefunction_label ='', index_convention = 'UpDown') -> FermionHamiltonian:
+        """
+        Loads the input state associated from this electronic structure problem with the corresponding label.
+
+        If `wavefunction_label` is not specified, it loads the greedy (Hartree-Fock) state.
+
+        `index_convention` can be 'UpDown' or 'HalfUp'
+        """
+        logger.info(f"Loading input state '{wavefunction_label}' from problem description using index_convention '{index_convention}'.")
+        args = { 'problem_description': self.__dict__, 'wavefunction_label': wavefunction_label, 'index_convention': index_convention }
+        args_json = json.dumps(map_tuples(args))
+        data = qsharp.client._execute(f'%chemistry.inputstate.load {args_json}', raise_on_stderr=True)
+        return InputState(data)
+
+
+class Broombridge(object):
+    """
+    Represents an instance of a broombridge schema data
+    """
+    def __init__(self, data: Dict):
+        self.__dict__ = data
+        # translate problem_descriptions into the actual datastructure:
+        self.problem_description = [ ProblemDescription(p) for p in data["problem_description"] ]
+
+    def __eq__(self, other): 
+        if not isinstance(other, FermionHamiltonian):
+            # don't attempt to compare against unrelated types
+            return NotImplemented
+        return self.__dict__ == other.__dict__
+
+
+def load_fermion_hamiltonian(file_name: str, index_convention = 'UpDown') -> FermionHamiltonian:
+    logger.info(f"Loading fermion hamiltonian from '{file_name}' using index_convention '{index_convention}'.")
+    """
+    Loads the fermion hamiltonian from the given file that contains broombridge data.
+
+    `index_convention` can be 'UpDown' or 'HalfUp'
+    """
+    args = { 'file_name': file_name, 'index_convention': index_convention }
+    args_json = json.dumps(map_tuples(args))
+    data = qsharp.client._execute(f'%chemistry.fh.load {args_json}', raise_on_stderr=True)
+    return FermionHamiltonian(data)
+
+
+def load_input_state(file_name: str, wavefunction_label : str, index_convention = 'UpDown') -> FermionHamiltonian:
```

Comment:
This suggestion aligns the signature with the comment that I left at 140, below.
```suggestion
def load_input_state(file_name: str, wavefunction_label : str = None, index_convention = 'UpDown') -> FermionHamiltonian:
```